Documentation/guides/messages/xa0035.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Xamarin.Android error XA0035
+description: XA0035 error code
+ms.date: 06/17/2020
+---
+# Xamarin.Android error XA0035
+
+## Issue
+
+Xamarin.Android was not able to determine the application's target
+[Android ABIs][abis] as specified by the `.csproj` file.
+
+[abis]: https://developer.android.com/ndk/guides/abis
+
+## Solution
+
+Make sure you are declaring valid values for `$(RuntimeIdentifiers)`:
+
+```xml
+<PropertyGroup>
+  <RuntimeIdentifiers>android.21-arm;android.21-arm64;android.21-x86;android.21-x64</RuntimeIdentifiers>
+</PropertyGroup>
+```
+
+See the Microsoft documentation on [runtime identifiers][rids] for more
+information.
+
+[rids]: https://docs.microsoft.com/dotnet/core/rid-catalog
```

Comment:
Probably it was already in mind when adding this info, but this link might need to be updated if the eventual page that talks about Android RIDs ends up being elsewhere. I think what's written here is good for now. I'm mostly just thinking out loud about the future.